src/components/PostDetails.tsx

```diff
@@ -132,7 +83,10 @@
                     type="button"
                     className="delete is-small"
                     aria-label="delete"
-                    onClick={() => deleteComment(comment.id)}
+                    onClick={() => {
+                      console.log(comment.id);
```

Comment:
The `console.log(comment.id);` statement is useful for debugging but should be removed or replaced with proper error handling or logging in production code.